docker-compose.yml

```diff
@@ -1,20 +1,27 @@
----
+version: '3.8'
 services:
-  postgres:
+  db:
     image: postgres:14
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
     environment:
-      POSTGRES_USER: ${DATABASE_USERNAME}
-      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
-      POSTGRES_DB: pet_rescue_development
-
-    ports:
-      - 5432
+      POSTGRES_DB: myapp_development
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
 
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready", "-U", "bwoa", "-d", "pet_rescue_development"]
-      interval: 10s
+  app:
+    build: .
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -b '0.0.0.0'"
     volumes:
-      - db:/var/lib/postgresql/data
+      - .:/myapp
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+    environment:
+      DATABASE_URL: postgres://postgres:password@db:5432/myapp_development
+      RAILS_ENV: development
 
 volumes:
-  db:
+  postgres-data:
+  bundle:
```

Comment:
you're not acutally using the bundle volume

Reply:
Ah ok.